cmd/image/qcow2ova/qcow2ova.go

```diff
@@ -159,10 +160,23 @@ Qcow2 images location:
 		if opt.ImageDist != "coreos" && opt.OSPassword == "" && !opt.OSPasswordSkip {
 			var err error
 			opt.OSPassword, err = GeneratePassword(12)
-			klog.Infof("Autogenerated OS root password is: %s", opt.OSPassword)
 			if err != nil {
 				return err
 			}
+			passwordfile, err := os.Create("password.txt")
+			if err != nil {
+				klog.Fatalf("failed to create password file, err: %v", err)
```

Comment:
Typically after `Fatalf`, os.Exit() is invoked, and hence the `return err` will not be reachable. In order to continue returning the error to the caller, we may use `Errorf`.
Ref: https://pkg.go.dev/k8s.io/klog#Fatalf
```suggestion
				klog.Errorf("failed to create password file, err: %v", err)
```